assets/src/components/v2/eink/departures_no_data.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ComponentType } from "react";
+import NoConnection from "Components/v2/bundled_svg/no_connection";
+
+interface Props {
+  show_alternatives: boolean;
+  stop_id: string;
+}
+
+const coolBlack = "#171F26";
```

Comment:
Same Q about allowable e-ink colors here

Reply:
Went back and double-checked the designs and this needs to be solid black `#000000`. Got that changed.